doc/source/getting_started.rst

```diff
@@ -87,24 +87,40 @@ CLI Options
 Usage: spyglass [OPTIONS]
 
 Options:
-  -s, --site TEXT                Specify the site for which manifests to be
-                                 generated
-  -t, --type TEXT                Specify the plugin type formation or tugboat
-  -f, --formation_url TEXT       Specify the formation url
-  -u, --formation_user TEXT      Specify the formation user id
-  -p, --formation_password TEXT  Specify the formation user password
-  -d, --additional_config PATH   Site specific configuraton details
-  -g, --generate_intermediary    Dump intermediary file from passed excel and
-                                 excel spec
-  -m, --generate_manifests       Generate manifests from the generated
-                                 intermediary file
-  -l, --loglevel INTEGER         Loglevel NOTSET:0 ,DEBUG:10,     INFO:20,
-                                 WARNING:30, ERROR:40, CRITICAL:50  [default:
-                                 20]
-  --help                         Show this message and exit.
-
-
-1. Running Spyglass with  Remote Data Source
+  -s, --site TEXT                 Specify the site for which manifests to be
```

Comment:
Should we create plugin as the sub command to spyglass, this will cleanup this help to an extent.

e.g.: spyglass [options] plugin_name [ options]
spyglass --loglevel 10 tugboat ...